product-scenarios/10-manage-security-resiliancy-other-aspects-of-service-endpoints/10.1-expose-service-with-secured-backend/10.1.1-expose-service-secured-via-basic-authentication/src/test/java/org/wso2/am/scenario/tests/basic/auth/security/BasicAuthEndpointSecuredAPITestcase.java

```diff
@@ -91,9 +90,13 @@ public void init() throws APIManagerIntegrationTestException {
         try {
             WebAppDeployUtils.deployWebApp(serviceEndpoint, admin, admin, warFileLocation,
                     warFileName);
+
+            boolean isWebAppDeployed =
+                    WebAppDeployUtils.isWebApplicationDeployed(serviceEndpoint, admin, admin, warFileName);
+            assertTrue(isWebAppDeployed, warFileName + " is not deployed" );
             log.info("WebApp deployed successfully");
         } catch (RemoteException | MalformedURLException e) {
-            e.printStackTrace();
+            log.error("Error when deploying webApp", e);
```

Comment:
Shall we print the failing web app name also in the log

Reply:
Added